tests/EStimLibrary.UnitTests/Core/SpatialModel/LocalizationTests.cs

```diff
@@ -0,0 +1,105 @@
+using EStimLibrary.Core.SpatialModel;
+using Xunit;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EStimLibrary.UnitTests.Core.SpatialModel
+{
+    /// <summary>
+    /// Test class for the LocalizationData record.
+    /// </summary>
+    public class LocalizationDataTests
+    {
+        /// <summary>
+        /// Test the basic construction of LocalizationData and 
+        /// ensure immutability (encapsulation).
+        /// </summary>
+        [Fact]
+        public void LocalizationData_Constructor()
```

Comment:
You can remove the class name prefix from each test method name. Just start with the name of the method you're testing (e.g., Constructor or Merge). See the "Our Coding Practices" notebook page > As You Go... section > item 4 for more test method naming convention description